client/src/pages/ConfirmationModal.tsx

```diff
@@ -0,0 +1,35 @@
+import './Modal.scss';
+import React, { useState } from 'react';
+import CommonModal from '../components/organisms/Modal';
+
+function ConfirmationModal(props: any) {
+  const [email, setEmail] = useState(props.email);
```

Comment:
We're not modifying the email in this component at all right? If so, I'd replace with 
```
const {email} = props
```